Formula/wso2ei.rb

```diff
@@ -0,0 +1,31 @@
+class Wso2ei < Formula
+  desc "WSO2 Enterprise Integrator"
+  homepage "https://wso2.com/integration/"
+  url "https://bintray.com/wso2/binary/download_file?file_path=wso2ei-6.4.0.zip"
```

Comment:
Please change this to `https://dl.bintray.com/wso2/binary/wso2ei-6.4.0.zip`, it follows the standard pattern that we use for Bintray URLs.